src/express-tooling.ts

```diff
@@ -33,37 +34,41 @@ const AUTHORIZATION_HEADER_FIELD_NAME = 'authorization';
  *  app.get('/path', requireScopesMiddleware['scopeA', 'scopeB'], (req, res) => { // Do route work })
  *
  * @param scopes - array of scopes that are needed to access the endpoint
+ * @param logger - optional logger
  * @param precedenceOptions - This options let consumers define a way to over rule scope checking. The parameter is optional.
  *
  * @returns { function(any, any, any): undefined }
  */
 function requireScopesMiddleware(scopes: string[],
+                                 logger?: Logger,
                                  precedenceOptions?: PrecedenceOptions): RequestHandler {
```

Comment:
The interface `PrecedenceOptions` still contains a non optional logger.

Reply:
which line? Can't see anything like that...

Reply:
Sorry my mistake, my IDE built a `d.ts`with old typing in the src folder